hypothesis-python/docs/reference/api.rst

```diff
@@ -1013,3 +1013,10 @@ For authors of test runners however, assigning to the ``inner_test`` attribute o
     and ``**kwargs`` expected by the original test.
 
 If the end user has also specified a custom executor using the ``execute_example`` method, it - and all other execution-time logic - will be applied to the *new* inner test assigned by the test runner.
+
+Detecting Hypothesis tests
+--------------------------
+
+To determine whether a test has been defined with Hypothesis or not, use |is_hypothesis_test|:
+
```

Comment:
I'd want to add a note: if you're working with `pytest`, our [pytest plugin] automatically marks all such tests, so you can use [`node.get_closest_marker("hypothesis")`](https://docs.pytest.org/en/stable/reference/reference.html#pytest.nodes.Node.get_closest_marker) rather than digging down to the callable.

Reply:
This is a not-found error:

```
:func:`node.get_closest_marker("hypothesis") <pytest:_pytest.nodes.Node.get_closest_marker>`
```

despite:

```
 ~/Desktop λ python -m sphinx.ext.intersphinx https://docs.pytest.org/en/stable/objects.inv | grep get_closest_marker
    _pytest.nodes.Node.get_closest_marker                                            : reference/reference.html#pytest.nodes.Node.get_closest_marker
 ~/Desktop λ 
```

and I gave up after 10 minutes, so no nice `get_closest_marker` link :(